app/templates/components/ui-table/cell/events/view/tickets/discount-codes/cell-actions.hbs

```diff
@@ -5,7 +5,7 @@
   {{#ui-popup content=(t 'Deactivate') class='ui icon button' position='left center'}}
     <i class="remove icon"></i>
   {{/ui-popup}}
-  {{#ui-popup content=(t 'Delete') class='ui icon button' position='left center'}}
+  {{#ui-popup content=(t 'Delete') click=(action (confirm (t 'Are you sure you would like to delete this Discount Code?') (action deleteDiscountCode record))) class='ui icon button' position='left center'}}
```

Comment:
@pradeepgangwar The component test is failing, you need to update the test file to accommodate the action you are passing.